auslib/web/admin/swagger/api.yaml

```diff
@@ -1597,74 +1963,79 @@ definitions:
 
   DataVersionModel:
     title: Data Version
-    description: version number of object
+    description: "model to re-use 'data_version' field."
```

Comment:
How about concat of both "model to re-use 'data_version' field. Data_version is the version  number of object" ?